packages/fuel-gauge/src/bytes.test.ts

```diff
@@ -32,54 +19,71 @@ type Wrapper = {
   inner_enum: SomeEnum;
 };
 
-const setup = async (balance = 500_000) => {
-  const provider = await Provider.create(FUEL_NETWORK_URL);
-  // Create wallet
-  const wallet = await generateTestWallet(provider, [[balance, BaseAssetId]]);
-
-  return wallet;
-};
-
 describe('Bytes Tests', () => {
-  let gasPrice: BN;
-  beforeAll(async () => {
-    const provider = await Provider.create(FUEL_NETWORK_URL);
-    ({ minGasPrice: gasPrice } = provider.getGasConfig());
-  });
-
   it('should test bytes output', async () => {
+    await using launched = await TestNodeLauncher.launch({
```

Comment:
I modified all `bytes.test.ts` tests to use `TestNodeLauncher`. You can run this test file without `--runInBand` and without a running node with this: `pnpm jest --no-cache packages/fuel-gauge/src/bytes.test.ts` and it'll work. The goal is to use this utility in all tests and remove `--runInBand` from the `pnpm test` command.

Reply:
Am I correct in assuming that anyone using this utility will have to be using at least TS 5.2?

Reply:
Yes, you're correct.